assets/styles/components/_wysiwyg.scss

```diff
@@ -0,0 +1,363 @@
+        & + h3,
+        & + h4,
+        & + h5,
+        & + h6 {
+            margin-top: size-clamp('md');
+        }
+    }
+
+    h2 {
+        // @include heading-h3;
+        margin-top: size-clamp('lg');
+        margin-bottom: size-clamp('md');
+
+        & + p,
+        & + ul,
+        & + ol,
+        & + table,
+        & + figure {
+            margin-top: size-clamp('md');
+        }
+
+        & + h1,
+        & + h2,
+        & + h3,
+        & + h4,
+        & + h5,
+        & + h6 {
+            margin-top: size-clamp('md');
+        }
+    }
+
+    h3 {
+        // @include heading-h4;
+        margin-top: size-clamp('lg');
+        margin-bottom: size-clamp('md');
+
+        & + p,
+        & + ul,
+        & + ol,
+        & + table,
+        & + figure {
+            margin-top: size-clamp('md');
+        }
+
+        & + h1,
+        & + h2,
+        & + h3,
+        & + h4,
+        & + h5,
+        & + h6 {
+            margin-top: size-clamp('md');
+        }
+    }
+
+    h4,
+    h5,
+    h6 {
+        margin-top: size-clamp('lg');
+        margin-bottom: size-clamp('sm');
+
+        & + p,
+        & + ul,
+        & + ol,
+        & + table,
+        & + figure {
+            margin-top: size-clamp('sm');
+        }
+
+        & + h1,
+        & + h2,
+        & + h3,
+        & + h4,
+        & + h5,
+        & + h6 {
+            margin-top: size-clamp('sm');
+        }
+    }
+
+    h4 {
+        // @include heading-h5;
+    }
+
+    h5 {
+        // @include heading-h6;
+    }
+
+    h6 {
+        // @include heading-h6;
+    }
+
+
+    // ==========================================================================
+    // Lists
+    // ==========================================================================
+    ul,
+    ol {
+        position: relative;
+        margin: size-clamp('md') 0;
+
+        >li {
+            padding-left: rem(30px);
+        }
+    }
+
+    ul {
+        & > li {
+            position: relative;
+
+            & + li {
+                margin-top: 0;
+            }
+
+            &::before {
+                content: "";
+                position: absolute;
+                left: 0;
+                top: 9px;
+                width: 8px;
+                height: 8px;
+                background-color: currentColor;
+                border-radius: 100%;
+            }
+        }
+    }
+
+    ol {
+        counter-reset: counter;
+
+        & > li {
+            counter-increment: counter;
+            position: relative;
+
+
+            & + li {
+                margin-top: 0;
+            }
+
+            &::before {
+                position: absolute;
+                left: 0;
+                content: counter(counter);
+            }
+        }
+    }
+
+    // ==========================================================================
+    // Link
+    // ==========================================================================
+    a {
+        display: inline;
```

Comment:
`display: inline;`

Why?

Reply:
In case we set another display globally for links, but I agree, we shouldn't. Property removed.